service/src/test/java/greencity/service/ubs/SuperAdminServiceImplTest.java

```diff
@@ -89,22 +87,16 @@ void addTariffServiceTest() {
         when(userRepository.findByUuid("123233")).thenReturn(user);
         when(locationRepository.findById(1L)).thenReturn(Optional.of(ModelUtils.getLocation()));
         when(bagRepository.save(bag)).thenReturn(bag);
-        when(bagTranslationRepository.saveAll(bag.getBagTranslations())).thenReturn(ModelUtils.getBagTransaltion());
 
         superAdminService.addTariffService(dto, "123233");
 
         verify(locationRepository).findById(1L);
         verify(bagRepository).save(bag);
-        verify(bagTranslationRepository).saveAll(bag.getBagTranslations());
     }
 
     @Test
     void getTariffServiceTest() {
-        when(bagTranslationRepository.findAll()).thenReturn(new ArrayList<>());
-
         superAdminService.getTariffService();
-
-        verify(bagTranslationRepository).findAll();
```

Comment:
there are no assertation or verification this test
Please fix it

Reply:
What does this test do?
You just call `superAdminService.getTariffService();` without any assertation or verification.